api/include/can_protocols/brake_can_protocol.h

```diff
@@ -59,13 +59,25 @@ typedef struct
                          Byte 0 should be \ref OSCC_MAGIC_BYTE_0.
                          Byte 1 should be \ref OSCC_MAGIC_BYTE_1. */
 
+#if defined(KIA_SOUL)
     uint16_t pedal_command; /*!< Pedal command. [65535 == 100%] */
 
     uint8_t enable; /*!< Command to enable or disable steering control.
-                     * Zero value means disable.
-                     * Non-zero value means enable. */
+                    * Zero value means disable.
+                    * Non-zero value means enable. */
 
     uint8_t reserved[3]; /*!< Reserved. */
+#elif defined(KIA_SOUL_EV)
+    uint16_t spoof_value_low; /*!< Value to be sent on the low spoof signal. */
+
+    uint16_t spoof_value_high; /*!< Value to be sent on the high spoof signal. */
+
+    uint8_t enable; /*!< Command to enable or disable steering control.
+                    * Zero value means disable.
+                    * Non-zero value means enable. */
```

Comment:
In all the other doxygen multi-line comments, the asterisk (`*`) characters are aligned to the first `*`, except here and lines 66/77.

Reply:
f308d1a